### PR TITLE
Added folder and file hands/test.py

### PR DESCRIPTION
**Pull request by Lolillosky**
Nada, solo es un test para aprender

Navidulaaaaa

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+log.txt
```
Comment (Maitane1972) [inline .gitignore:1]: Please, make it more generic.

*.txt

File: hands/test.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import matplotlib.pyplot as plt
+
+path = 'D:/Hand data/multiview_hand_pose_dataset_uploaded_v2/'
+
+images_folder = 'annotated_frames/'
+bound_folder = 'bounding_boxes/'
+
+fig,ax = plt.subplots(1)
+
+for i in range(500):
```
Comment (Maitane1972) [inline hands/test.py:15]: Please, leave 100 phots

File: .gitignore (hunk 1 above)
Comment (Lolillosky) [reply .gitignore:1]: Ok, I will change it

File: hands/test.py (hunk 2 above)
Comment (Lolillosky) [reply hands/test.py:15]: No problema

**Comment by Lolillosky** [comment]
Everything resolved!!!!

Please, take another look

File: hands/test.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        y =  0.5*(box.loc['TOP'][1] + box.loc['BOTTOM'][1])
+        plt.cla()
+
+    # Display the image
+        ax.imshow(image)
+
+        rest = patches.Rectangle((box.loc['LEFT'][1], box.loc['TOP'][1]), box.loc['RIGHT'][1]-box.loc['LEFT'][1], box.loc['BOTTOM'][1] - box.loc['TOP'][1], 
+            linewidth=1,edgecolor='r', fill = False)
+
+        circ = patches.Circle((x,y),10,linewidth=1,edgecolor='r',facecolor='r')
```
Comment (Maitane1972) [inline hands/test.py:43]: Please, I want no fill

Comment (Maitane1972) [inline hands/test.py:43]: i want no fill

File: hands/test.py [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    try:
+        data = np.random.randint(1, 21)
+        n1 = np.random.randint(0, 200)
+        n2 = np.random.randint(1,4)
+
+        folder_name = 'data_' + str(data) + '/'
+        image_file_name = str(n1) + '_webcam_' + str(n2) + '.jpg'
+        box_file_name = str(n1) + '_bbox_' + str(n2) + '.txt'
+
+        image = np.array(Image.open(path + images_folder + folder_name + image_file_name))
```
Comment (Maitane1972) [inline hands/test.py:26]: ooops

**Comment by Lolillosky** [comment]
Evrything ready

**Pull request merged**